tests/chain_tests/delay_tests.cpp

```diff
@@ -923,15 +923,19 @@ BOOST_AUTO_TEST_CASE( link_delay_link_change_test ) { try {
    liquid_balance = get_currency_balance(chain, N(tester2));
    BOOST_REQUIRE_EQUAL(asset::from_string("0.0000 CUR"), liquid_balance);
 
-   try {
+   BOOST_REQUIRE_EXCEPTION(
       chain.push_action(config::system_account_name, contracts::linkauth::get_name(), tester_account, fc::mutable_variant_object()
       ("account", "tester")
       ("code", "currency")
       ("type", "transfer")
       ("requirement", "second"),
-      30, 3);
-      BOOST_FAIL("linkauth should have a min delay set by previous linkauth");
-   } catch (...) { }
+      30, 3),
+      transaction_exception,
+      [] (const transaction_exception &e)->bool {
+         BOOST_REQUIRE_EQUAL(std::string("transaction validation exception"), e.what());
```

Comment:
Please use  e.to_detail_string() and check for something like "greater than the delay specified" so we know that it is this exception, same for the other BOOST_REQUIRE_EXCEPTIONs that are similar to this.